eyewear_shop/demo/calendar_event.xml

```diff
@@ -0,0 +1,85 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+    <record id="calendar_event_1" model="calendar.event">
```

Comment:
Shouldn't there be start and end hours? Or do we accept they're coming whenever they want during the day?

Reply:
Yes we can add start and end hours,  thanks for the suggestion